docs/SetupDatabaseForAWS.md

```diff
@@ -1,30 +1,151 @@
+
+You can configure the **Read/Write Capacity** of DynamoDB tables based on your requirements. Please refer to the official document for more details on Read/Write Capacity.
+
+* [Read/write capacity mode](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html)
+
+You can configure Read/Write Capacity using ScalarDB/DL Schema Loader when you create a table. Please refer to the following document for more details on how to configure Read/Write Capacity (RU) using ScalarDB/DL Schema Loader.
+
+* [ScalarDB Schema Loader](https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader.md)
+
+## Amazon RDS for MySQL, PostgreSQL, Oracle, and SQL Server
+
+### Authentication method
+
+When you use RDS, you must set `JDBC_URL`, `USERNAME`, and `PASSWORD` in the ScalarDB/ScalarDL properties file as follows.
+
+```properties
+scalar.db.contact_points=<JDBC_URL>
+scalar.db.username=<USERNAME>
+scalar.db.password=<PASSWORD>
+scalar.db.storage=jdbc
+```
+
+Please refer to the following document for more details on the properties for RDS (JDBC databases).
+
+* [Getting Started with ScalarDB on JDBC databases](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-jdbc.md)
+
+### Required configuration/steps
+
+#### Create an RDS database instance
+
+You must create an RDS database instance. Please refer to the official document for more details.
+
+* [Configuring an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_RDS_Configuring.html)
+
+### Optional configurations/steps
+
+#### Enable automated backups (Recommended in the production environment)
+
+You can enable automated backups. Please refer to the official document for more details.
+
+* [Working with backups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithAutomatedBackups.html)
+
+#### Configure monitoring (Recommended in the production environment)
+
+You can configure the monitoring and logging of RDS using its native feature. Please refer to the official documents for more details.
+
+* [Monitoring metrics in an Amazon RDS instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html)
+* [Monitoring events, logs, and streams in an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitor_Logs_Events.html)
+
+#### Disable public access (Recommended in the production environment)
+
+Public access is disabled by default. You can access the RDS database instance from the Scalar product pods on your EKS cluster as follows.
+
+* Create the RDS database instance on the same VPC as your EKS cluster.
+* Connect the VPC for the RDS and the VPC for the EKS cluster for the Scalar product deployment using [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html). (// TODO: We need to test this feature with Scalar products.)
```

Comment:
We have not yet tested this feature with Scalar products.
I will do the test and update the document as another task.